dvc/progress.py

```diff
@@ -33,43 +35,72 @@ class Tqdm(tqdm):
     maximum-compatibility tqdm-based progressbars
     """
 
+    BAR_FMT_DEFAULT = (
+        "{percentage:3.0f}%|{bar:10}|{desc} {bar:-10b}{n}/{total}"
+        " [{elapsed}<{remaining}, {rate_fmt:>11}{postfix}]"
+    )
+    BAR_FMT_NOTOTAL = (
+        "{desc} {bar:b}{n} [{elapsed}<??:??, {rate_fmt:>11}{postfix}]"
+    )
+
     def __init__(
         self,
         iterable=None,
         disable=None,
-        bytes=False,  # pylint: disable=W0622
+        level=logging.ERROR,
+        desc=None,
         desc_truncate=None,
         leave=None,
+        level_leave=logging.DEBUG,
```

Comment:
What is the whole point of this `level_leave`?  Why do we leave pbars when we are showing debugging output?

Reply:
I think a completed pbar would seem to be at the same "level" as debug messages (especially when it has a descriptive title). @efiop?

Reply:
@casperdcl You mean that we will leave progress bar traced at debug level but not at info? That seems weird. I don't think we need them on debug at all.

Reply:
yes, that's what I meant. it just seems that the debug output is so much at the moment that an extra couple of progressbars wouldn't be an issue. can remove if you prefer.

Reply:
@casperdcl Yeah, but are those progress bars even useful in the debug output? I don't think so, to be honest. Progress bars are more about keeping the user up to date, for me, as a developer, dvc has proper debug messages in places where they are actually needed. I think it would be better to remove those.